pycbc/inference/models/marginalized_gaussian_noise.py

```diff
@@ -262,7 +274,12 @@ def _loglr(self):
         """
         params = self.current_params
         try:
-            wfs = self.waveform_generator.generate(**params)
+            if self.all_ifodata_same_rate_length:
```

Comment:
Marginalized phase might also need this bit.

Reply:
Added for marginalized phase